### ISSUE TITLE
Unhandled SEHException nLog 4.7.5 NLog.Targets.FileTarget.CleanupInitializedFiles()

### ISSUE THREAD

**Issue by ngrinschgl** [NLog/NLog#4550]
Hi all!
I have an app running/logging with nLog 4.7.5 and it ran fine for months now (uptime 80%). This week
the app crashed and we found the following stack in the windows event logs.

SomeApp.exe
Framework Version: v4.0.30319
Description: The process was terminated due to an unhandled exception.
Exception Info: System.Runtime.InteropServices.SEHException
```
at NLog.Targets.FileTarget.CleanupInitializedFiles()
at NLog.Targets.FileTarget.InitializeFile(System.String, NLog.LogEventInfo)
at NLog.Targets.FileTarget.ProcessLogEvent(NLog.LogEventInfo, System.String, System.ArraySegment`1<Byte>)
at NLog.Targets.FileTarget.Write(NLog.LogEventInfo)
at NLog.Targets.Target.Write(NLog.Common.AsyncLogEventInfo)
at NLog.Targets.Target.WriteAsyncThreadSafe(NLog.Common.AsyncLogEventInfo)
at NLog.Targets.Target.WriteAsyncLogEvent(NLog.Common.AsyncLogEventInfo)
at NLog.LoggerImpl.WriteToTargetWithFilterChain(NLog.Targets.Target, NLog.Filters.FilterResult, NLog.LogEventInfo, NLog.Common.AsyncContinuation)
at NLog.LoggerImpl.Write(System.Type, NLog.Internal.TargetWithFilterChain, NLog.LogEventInfo, NLog.LogFactory)
at NLog.Logger.Debug(System.String)
at SomeApp.MainForm.InsertMessage(System.String)
```

Calling method:
```
 public void InsertMessage(string text)
            {
                try
                {
                    if (lbLog.InvokeRequired)
                    {
                        InsertMessageCallback d = InsertMessage;
                        Invoke(d, text);
                    }
                    else
                    {
                        _log_.Debug(text);
                    }
                }
                catch (Exception ex)
                {
                    _log_.Fatal(ex, "oh snap");
                }
            }

```

Any idea what could cause this exception?

Thanks!

#nlog

**Comment by snakefoot** [comment]
[SEHException](https://docs.microsoft.com/en-us/dotnet/api/system.runtime.interopservices.sehexception) means exception from native unmanaged code. Cannot see any native code in the callstack shown here. So I don't have any good ideas.

**Comment by snakefoot** [comment]
Closing as resolved with  #4551
